Resources/Prototypes/_Sunrise/Recipes/Lathes/security.yml

```
@@ -285,6 +285,16 @@
   materials:
     Steel: 193
 
+- type: latheRecipe
+  parent: BaseWeaponRecipeSidearm
+  id: FlareGunSecurityUpgrade
+  result: FlareGunSecurityUpgrade
+  categories:
+  - Weapons
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)